draft-ietf-tls-tls13.md

```diff
@@ -5636,6 +5762,69 @@ application protocols separately ensuring that confidential
 information is not inadvertently leaked.
 
 
+## Replay Attacks on 0-RTT {#replay-0rtt}
+
+Replayable 0-RTT data presents a number of security threats to
+TLS-using applications. Specifically, if applications are not
+engineered to be replay safe (minimally, this means idempotent, but in
+some cases may require even stronger conditions), then duplication of
+actions may cause side effects (e.g., purchasing an item or
+transferring money) to be duplicated, thus harming the site or the
+user.  Attackers can also store and replay 0-RTT messages in order to
+re-order them with respect to other messages, which may harm
+unsuspecting applications.
+
+In addition, if data can be replayed a large number of times,
+additional attacks become possible. Specifically, attackers
+can use multiple replays to exploit information leakage via
+side channels such as timing network caches or measuring the
+speed of cryptographic. In addition, they may be able to
+overload rate-limiting systems. For further description of
+these attacks, see {{Mac17}}.
+
+Ultimately, servers have the responsibility to protect themselves
+against attacks employing 0-RTT data replication. The mechanism
+described in {{anti-replay}} are intended to
+prevent replay at the TLS layer do not provide complete protection
+against receiving multiple copies of client data:
+TLS 1.3 falls back to the 1-RTT
+handshake when the server does not have any information about the
+client, e.g., because it is in a different cluster which does not
+share state or because the ticket has been deleted as described in
+{{single-use-tickets}}. If the application layer protocol retransmits
+data in this setting, then it is possible for an attacker to induce a
+replay attack by sending the ClientHello to both the original cluster
+(which processes the data immediately) and another cluster which will
+fall back to 1-RTT and process the data upon application layer
+replay. The scale of this attack is limited by the client's
+willingness to replay and therefore only allows a limited number of
+replays, which will also use different encryption keys.
```

Comment:
"limited amount of duplication."  These aren't replays.

Also, drop the "which will also use different encryption keys", it's not relevant.